CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+Contributing
+============
+We gratefully accept bug reports and contributions from the community. There are some requirements we need to fulfill in order to be able to integrate contributions:
+
+ - As with any open source project, contributions will be reviewed by the project team and community and may need some modifications to be accepted.
+ - The contribution should not break API or ABI, unless there is a real justification for that. If there is an API change, the contribution, if accepted, will be merged only when there will be a major release.
+
+Contributor License Agreement (CLA)
+-----------------------------------
+- All contributions, whether large or small, require a Contributor's License Agreement (CLA) to be accepted. This is because source code can possibly fall under copyright law and we need your consent to share in the ownership of the copyright.
```

Comment:
Consider removing "possibly".

Reply:
Consider leaving it in, since it's legally correct.

Reply:
"Can" already implies "possibly". This change wouldn't change the meaning.

Reply:
It doesn't change the meaning, only changes the level of doubt someone might have. We don't want doubt here, so better to remove it. I'd also suggest strengthening it as:

> source code falls under copyright law